src/databricks/labs/ucx/workspace_access/clusters.py

```diff
@@ -0,0 +1,119 @@
+        return DataSecurityMode.USER_ISOLATION
+
+    def map_cluster_to_uc(self, cluster_id: str):
+
+        cluster_id_list = self._get_cluster_id(cluster_id)
+        spark_version = self._ws.clusters.select_spark_version(latest=True, long_term_support=True)
+        for cluster in cluster_id_list:
+            try:
+                cluster_details = self._ws.clusters.get(cluster)
+                if cluster_details.data_security_mode is None:
+                    raise InvalidParameterValue(f"Data security Mode is None for the cluster {cluster}")
+                access_mode = self._get_access_mode(cluster_details.data_security_mode.name)
+                self._installation.save(cluster_details, filename=f'backup/clusters/{cluster_details.cluster_id}.json')
+                logger.info(f"Editing the cluster {cluster} with access_mode as {access_mode}")
+                self._ws.clusters.edit(
+                    cluster_id=cluster,
+                    cluster_name=cluster_details.cluster_name,
+                    spark_version=spark_version,
+                    num_workers=cluster_details.num_workers,
+                    spark_conf=cluster_details.spark_conf,
+                    spark_env_vars=cluster_details.spark_env_vars,
+                    data_security_mode=access_mode,
+                    node_type_id=cluster_details.node_type_id,
+                    autoscale=cluster_details.autoscale,
+                    policy_id=cluster_details.policy_id,
+                    autotermination_minutes=cluster_details.autotermination_minutes,
+                    custom_tags=cluster_details.custom_tags,
+                    init_scripts=cluster_details.init_scripts,
+                    cluster_log_conf=cluster_details.cluster_log_conf,
+                    aws_attributes=cluster_details.aws_attributes,
+                    ssh_public_keys=cluster_details.ssh_public_keys,
+                    enable_elastic_disk=cluster_details.enable_elastic_disk,
+                    cluster_source=cluster_details.cluster_source,
+                    instance_pool_id=cluster_details.instance_pool_id,
+                    enable_local_disk_encryption=cluster_details.enable_local_disk_encryption,
+                    driver_instance_pool_id=cluster_details.driver_instance_pool_id,
+                )
+            except InvalidParameterValue as e:
+                logger.warning(f"skipping cluster remapping: {e}")
+
+    def revert_cluster_remap(self, cluster_ids: str, total_cluster_ids: list):
+        if cluster_ids != "<ALL>":
+            cluster_list = [x.strip() for x in cluster_ids.split(",")]
+        else:
+            cluster_list = total_cluster_ids
+        logger.info(f"Reverting the configurations for the cluster {cluster_list}")
+        for cluster in cluster_list:
+            try:
+                cluster_details = self._installation.load(ClusterDetails, filename=f"/backup/clusters/{cluster}.json")
```

Comment:
```suggestion
                cluster_details = self._installation.load(ClusterDetails, filename=f"backup/clusters/{cluster}.json")
```
remove first forward slash. it'll confuse the installation

Reply:
Removed the forward slash